src/test/java/plugins/PrioritySorterPluginTest.java

```diff
@@ -34,6 +36,7 @@
 
     @Before
     public void setUp() throws Exception {
+        assumeTrue("This test requires a restartable Jenkins", jenkins.canRestart());
```

Comment:
this would ideally be a rule so that we can avoid the overhead of starting Jenkins for a test that is doomed to failure - but how do we know if we can restart ahead of time :-(